src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/HmppsDomainEventsEmitterIntTest.kt

```diff
@@ -204,22 +204,12 @@ class HmppsDomainEventsEmitterIntTest : QueueIntegrationTest() {
     await untilCallTo { getNumberOfMessagesCurrentlyOnQueue() } matches { it == 0 }
 
     // expecting 3 attempts at messages - the initial create then 2 updates
```

Comment:
This comment is now incorrect as you are resetting the mock - but possibly not needed anymore because I think the 2 messages are self explanatory

Reply:
OK, I have one more minor PR to do so I with fix that